tests/xml.js

```diff
@@ -150,6 +152,18 @@ define([
             eq('<output attr=value />', '<output attr="value" />', false);
         });
 
+        it("should fail on escape unquoted attribute value with <", function () {
+            chai.expect(function () {
+                eq('<output value=2<3 />', '<output value="2&lt;3" />', false);
```

Comment:
What would `normalize` or `humanize` return? or is this just saying that they won't match each other?